### PR TITLE
Added Cerberus check to job and update CRDs

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves [OCPQE-12293](https://issues.redhat.com/browse/OCPQE-12293)

### CODE REVIEW

**Comment by paigerube14** [comment]
From the cerberus addition perspective, things look good to me. Will let Mehul comment on the other file changes

**Comment by nathan-weinberg** [comment]
Retested here -  see build artifacts for the metadata JSON: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/172/

File: scripts/flows_v1alpha1_flowcollector_lokistack.yaml [hunk 1]
```diff
@@ -60,8 +68,7 @@ spec:
         certFile: user.crt
         certKey: user.key
   loki:
-    url: https://lokistack-distributor-http.openshift-operators-redhat.svc:3100/
-    querierUrl: https://lokistack-querier-http.openshift-operators-redhat.svc:3100/
+    url: 'http://loki.netobserv.svc:3100/'
```
Comment (jotak) [inline scripts/flows_v1alpha1_flowcollector_lokistack.yaml:71]: is this change intended?
